RecoLocalCalo/EcalRecProducers/plugins/Common.h

```diff
@@ -0,0 +1,46 @@
+#ifndef RecoLocalCalo_EcalRecProducers_plugins_Common_h
+#define RecoLocalCalo_EcalRecProducers_plugins_Common_h
+
+#include <cassert>
+#include <chrono>
+#include <cmath>
+#include <cstdint>
+#include <iostream>
+
+#include <cuda.h>
+#include <cuda_runtime.h>
+
+// a workaround for std::abs not being a constexpr function
+namespace ecal {
+
+  template <typename T>
+  constexpr T abs(T const& value) {
+    return ::std::max(value, -value);
+  }
+
+  // temporary
+  namespace mgpa {
+
+    constexpr int adc(uint16_t sample) { return sample & 0xfff; }
+    constexpr int gainId(uint16_t sample) { return (sample >> 12) & 0x3; }
+
+  }  // namespace mgpa
+
+}  // namespace ecal
+
+template <typename T>
+struct DurationMeasurer {
```

Comment:
this does not seem specific to ECAL -> put in a common GPU utils place.

Reply:
I think this is not used any more, so it could just be dropped.